hack/setup-temporary-gopath.sh

```diff
@@ -8,7 +8,7 @@ set -o nounset
 # the current repo directory is not within GOPATH.
 function shim_gopath() {
   local REPO_DIR=$(git rev-parse --show-toplevel)
-  local TEMP_GOPATH="${REPO_DIR}/.gopath"
+  local TEMP_GOPATH="/tmp/.gopath"
```

Comment:
```suggestion
  local TEMP_GOPATH="$(mktemp -d)/.gopath"
```

Reply:
and in other places

Reply:
Same comment as @dprotaso, using `mktemp` is "better" 👼🏼